### ISSUE TITLE
Cannot access Storage anymore (work fine with previous version)

### ISSUE THREAD

**Issue by mbnshtck** [Azure/azure-ruby-asm-core#22]
using new version 0.1.3 I get HTTP 403 error, trying to access a table. 
Work OK in with version 0.1.2

**Comment by edwardvalentini** [comment]
+1 same exact ssue here... 
```
Azure::Core::Http::HTTPError: AuthenticationFailed (403): Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.
```

**Comment by stevenmaguire** [comment]
I am also experiencing this behavior. I have provided valid account name and access key, outgoing request includes a, what appears to be, valid Authorization header and the response is 403. Did something change?

**Comment by veronicagg** [comment]
Thanks for reporting this issue, we're investigating. Could you share an extract of the code that's giving the error? 
@yaxia Please share your findings on this issue.

**Comment by yaxia** [comment]
I've reverted the breakings and released 0.1.4. Please update. Sorry for the inconvenience.

**Comment by edwardvalentini** [comment]
thanks @yaxia its working great now!
